colorbleed/lib.py

```diff
@@ -245,3 +245,23 @@ def collect_container_metadata(container):
         return {}
 
     return hostlib.get_additional_data(container)
+
+
+def get_project_fps():
+    """Returns project's FPS, if not found will return 25 by default
+
+    Returns:
+        float
+    """
+
+    project_name = io.active_project()
+    project = io.find_one({"name": project_name,
+                           "type": "project"},
+                          projection={"config": True})
+
+    config = project.get("config", None)
+    assert config, "This is a bug"
+
+    fps = config.get("fps", 25.0)
+
+    return float(fps)
```

Comment:
Because you're forcing it to a float here it means that the `set_project_fps` will never actually trigger with and integer, not a problem - just wanted to state that. :)

Reply:
Might be we need to look at creating an FPS table. The following is occuring:
When your FPS is 25 in the scene, it displays as an integer. Using `mel.eval('currentTimeUnitToFPS()')` returns a float so the 25 FPS becomes 25.0. Maybe a float / int comparisonneeds to be added so we return whatever FPS has been set.

Reply:
The issue just got more annoying, running `currentTimeUnitToFPS` in mel it returns the correct values. When it's a float it returns a float, it its an int. Running the code in python through `mel.eval` always returns a float, It doesn't properly return the original value. Maybe you know why?